VAMobile/src/styles/themes/VAColors.ts

```diff
@@ -76,4 +76,6 @@ export default {
   darkmodeRed: '#FF4F5B',
   darkmodeGrayWarmDark: '#292929',
   primaryDarker2: '#2864AF',
+  // USWDS colors
+  uswdsRedVivid40: '#fb5a47',
```

Comment:
Wouldn't be needed at all using the design system Button component (already in the package level the app is at), but perhaps the app should consider adding the [tokens package](https://www.npmjs.com/package/@department-of-veterans-affairs/mobile-tokens) if manually adding it to get the ball rolling on shifting colors over, although I think there may be a specific ticket tracking that.

Reply:
Nice callout on the tokens package. I'll add that in, especially since we'll need the other colors from there down the line